### ISSUE TITLE
[All version][Web]Error is not show if you try to propose more that one law at a time.

### ISSUE THREAD

**Issue by elfl0rd** [StrangeLoopGames/EcoIssues#8421]
So, as i see - if you not admin you can not propose more that one law at a time.
You can add new law only after first law is approved\rejected.
But...
You can not see correct error message, only "internal server error" in console and empty red bar.
![default](https://user-images.githubusercontent.com/4980243/42516873-326a2fb8-8467-11e8-9164-891060c0fc00.png)
(one proposed law - my, "test")
![default](https://user-images.githubusercontent.com/4980243/42517020-77b42042-8467-11e8-9054-39c875e75960.png)
![default](https://user-images.githubusercontent.com/4980243/42517058-8812d2a8-8467-11e8-84c9-f9edc519f681.png)

**Comment by MarjieAVolk** [comment]
Partial duplicate of https://github.com/StrangeLoopGames/EcoIssues/issues/7990

The limit on how many laws you can propose was an effort to restrict griefing, I believe.  The game has changed a bit since then though; new players can't propose laws by default now, and also it is possible to propose as many clauses as you want in a single law, so the restriction doesn't really accomplish anything.  So yes, I think we can remove it.

**Comment by elfl0rd** [comment]
yep, duplicate...
Restrict to one law from player - good.
2h is not problem.
But need
1) Show correct error.
2) Ability to allow server leader or all to make multiple laws.
Like 
After this law passed, nonadmin server leader can create multiply law at one time, or 
After this law passed, any player can create multiply law at one time!

**Comment by MarjieAVolk** [comment]
Oh you want it to be like a votable setting?  Interesting.

**Comment by elfl0rd** [comment]
Yes, make players more free to control government. But if it is hard to make - please allow leader (like as admin) and maybe show right error :P

**Comment by MarjieAVolk** [comment]
In 8.0, everyone will be allowed to propose multiple laws
